pom.xml

```diff
@@ -488,10 +564,25 @@
         <plugin>
           <groupId>org.scalastyle</groupId>
           <artifactId>scalastyle-maven-plugin</artifactId>
-          <version>1.0.0</version>
+          <version>${scalastyle.version}</version>
           <configuration>
-            <configLocation>${project.basedir}/scalastyle-config.xml</configLocation>
+            <verbose>false</verbose>
+            <failOnViolation>true</failOnViolation>
+            <includeTestSourceDirectory>true</includeTestSourceDirectory>
+            <failOnWarning>false</failOnWarning>
+            <sourceDirectory>${project.basedir}/src/main/scala</sourceDirectory>
+            <testSourceDirectory>${project.basedir}/src/test/scala</testSourceDirectory>
+            <outputFile>${project.basedir}/scalastyle-output.xml</outputFile>
+            <outputEncoding>UTF-8</outputEncoding>
           </configuration>
+          <executions>
+            <execution>
+              <phase>validate</phase>
+              <goals>
+                <goal>check</goal>
+              </goals>
+            </execution>
+          </executions>
         </plugin>
```

Comment:
Any reason we will still keep the usage of scalastyle while scalafmt is being introduced?